docs/esp32/quickref.rst

```diff
@@ -833,9 +833,9 @@ The RMT is ESP32-specific and allows generation of accurate digital pulses with
     import esp32
     from machine import Pin
 
-    r = esp32.RMT(0, pin=Pin(18), clock_div=8)
-    r   # RMT(channel=0, pin=18, source_freq=80000000, clock_div=8)
-    # The channel resolution is 100ns (1/(source_freq/clock_div)).
+    r = esp32.RMT(pin=Pin(18), resolution_hz=10000000)
+    r   # RMT(pin=18, source_freq=80000000, clock_div=8)
```

Comment:
I think this will print out the `resolution_hz` when you do `print(r)`, so would be good to include that info here, to match the true output.